openhcl/underhill_attestation/src/igvm_attest/mod.rs

```diff
@@ -36,19 +33,45 @@ pub enum Error {
     },
 }
 
+/// Rust-style enum for `IgvmAttestReportType`
+pub enum ReportType {
+    /// VBS report
+    // TODO VBS
+    #[allow(dead_code)]
```

Comment:
use `expect(dead_code)` instead of `allow(dead_code)`, so that the Rust compiler warns you if this lint supression ever stops being necessary